components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/flow/impl/flows/backchain/v1/TransactionBackchainReceiverFlowV1.kt

```
@@ -42,9 +48,16 @@ class TransactionBackchainReceiverFlowV1(
 
         val sortedTransactionIds = TopologicalSort()
 
+        val ledgerConfig = flowConfigService.getConfig(UTXO_LEDGER_CONFIG)
+
+        requireNotNull(ledgerConfig) {
+            "Could not find UTXO ledger config in the flow context with key: $UTXO_LEDGER_CONFIG"
+        }
```

I think this require should be in `getConfig`. For now, lets just make the assumption that all config either exists with a default value or should throw an error upon not being found.

Otherwise things look fine to me.